.gitignore

```diff
@@ -36,3 +36,6 @@ venv*/
 /frontend/node_modules
 pip-wheel-metadata
 site
+
+# Temporary bundle data
+codalab-worker-scratch
```

Comment:
Can you explain what this bundle data is used for?

Reply:
https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/main.py#L38